### PR TITLE
New bugs found

### PR DESCRIPTION
**Pull request by QuantumAli**
**Done on this branch**
- Fixed certain issues on tags with the grants (removing additional duplicate tags)
- Removed ** from title

**Other Bugs Found**
- Also the slider for BlogSlider is not working, as of now it only looks at "Unitary Fund 2021 Annual Report"
- If it is possible to have tags under grants to be all lowercase. Some tags were written with uppercase, and this creates an additional tag on when filtering through

### CODE REVIEW

File: src/content/grant/2021_Quantum_algorithms.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-name: "Quantom Alogirthms"
+name: "Quantom Alogrithms"
```
Comment (WrathfulSpatula) [inline src/content/grant/2021_Quantum_algorithms.md]: "Quantum Algorithms"

File: src/content/grant/2021_Quantum_algorithms.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 year: 2021
 month: 3
 day: 10
 country: SG
 tags:
   - quantum education
-  - quantum alogrithms
+  - quantum alogrithm
```
Comment (WrathfulSpatula) [inline src/content/grant/2021_Quantum_algorithms.md]: "quantum algorithm"

**Review by WrathfulSpatula — CHANGES_REQUESTED** [review]
Please fix the two typos, and then I'll approve.

**Comment by QuantumAli** [comment]
Fixed them!

**Review by WrathfulSpatula — APPROVED** [review]

**Review by nathanshammah — APPROVED** [review]
Excellent fixes and finds @QuantumAli! Can you please open two additional issues on GH about the things you found out?

**Pull request merged**